ocs_ci/utility/framework/fusion_fdf_init.py

```diff
@@ -200,8 +201,8 @@ def generate_run_id() -> int:
         int: Unique identifier for the run
 
     """
-    logger.debug("Generating run_id from timestamp")
-    run_id = int(time.time())
+    logger.debug("Generating run_id from timestamp and random number")
+    run_id = int(time.time()) * 1000 + random.randint(0, 1000)
```

Comment:
```suggestion
    run_id = int(time.time() * 1000)
```
As time.time() returns something like:
1746544165.797612

You can make it quite easy with milliseconds included instead of using random. Then the chance we hit the same millisecond is probably impossible